### PR TITLE
Version 3.43

### PR DESCRIPTION
**Pull request by khwilliamson**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #---------------------------------------------------------------------------
 3.42 2020-11-10 Karl Williamson <khw@cpan.org>
 	- Drop EUMM version prereq, Graham Knop++
-3.41 2020-10-09 Karl Williamson <khw@cpan.org>
+3.42 2020-10-09 Karl Williamson <khw@cpan.org>
```
Comment (karenetheridge) [inline ChangeLog:9]: This looks like a misapplied search-and-replace. But I'm not sure why this commit is here at all.

File: ChangeLog [hunk 2]
```diff
@@ -1,5 +1,9 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.43 2021-06-15 Karl Williamson <khw@cpan.org>
```
Comment (karenetheridge) [inline ChangeLog:3]: 3.43 hasn't been released yet, so this date is wrong?  I'd just omit the date until it's time to release.

File: ChangeLog [hunk 3]
```diff
@@ -1,5 +1,7 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.42 2020-11-10 Karl Williamson <khw@cpan.org>
```
Comment (karenetheridge) [inline ChangeLog:7]: ditto, re 3.42

File: lib/Pod/Simple.pm [hunk 4]
```diff
@@ -18,7 +18,7 @@ use vars qw(
 );
 
 @ISA = ('Pod::Simple::BlackBox');
-$VERSION = '3.40';
+$VERSION = '3.41';
```
Comment (karenetheridge) [inline lib/Pod/Simple.pm]: ditto, re 3.41

File: ChangeLog [hunk 5]
```diff
@@ -1,5 +1,12 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.41 2020-10-09 Karl Williamson <khw@cpan.org>
```
Comment (karenetheridge) [inline ChangeLog]: Why is this commit here? Hasn't 3.41 already been released?

File: ChangeLog [hunk 6]
```diff
@@ -1,5 +1,9 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.43 2021-06-15 Karl Williamson <khw@cpan.org>
+	- Remove pod markup from README
+	- Update homepage to metacpan
+	- "use" not "require" Pod::Simple::TiedOutFH
```
Comment (karenetheridge) [inline ChangeLog:6]: I'd add a reference to the PR here: +`(PR#131, Jim Keenan)`

**Review by karenetheridge — CHANGES_REQUESTED** [review]
I'm a little confused about some of the commits in this PR.  Since they are already in master, there is no need to apply them again - they can be rebased in. `git fetch --all; git rebase -i origin/master`

File: ChangeLog (hunk 5 above)
Comment (khwilliamson) [reply ChangeLog:9]: Apparently, the administrative changes didn't get pushed to master at the proper time.

File: ChangeLog (hunk 1 above)
Comment (khwilliamson) [reply ChangeLog]: I believe this has now been corrected.

File: ChangeLog (hunk 5 above)
Comment (khwilliamson) [reply ChangeLog:9]: Things have been fixed up as a result of this comment

**Comment by khwilliamson** [comment]
This has been released through means other than this pull request

**Comment by haarg** [comment]
The master branch in this repo still seems to be out of date.

**Comment by khwilliamson** [comment]
On 7/26/21 2:46 PM, Graham Knop wrote:
> The master branch in this repo still seems to be out of date.

Thanks

**Pull request closed without merging**